src/pages/RealEstateDetail/ControllerWrap/ControllerWrap.js

```diff
@@ -1,53 +1,74 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import styled from 'styled-components';
 
-function ControllerWrap() {
+function ControllerWrap(props) {
+  console.log(props.dealInvestmentOption);
+  const investmentOption = number => {
+    let result = '';
+    let array = [];
+    for (let i = 0; i < number.length; i++) {
+      result = number[i] / 10000;
+      array.push(result);
+    }
+    return array;
```

Comment:
```suggestion
  const investmentOption = numbers => {
    return numbers.map(number => number / 10000);
```
Array 메소드 잘 활용하셔야 합니다~ map 활용해서 바로 할 수 있어보입니다!

Reply:
for문으로 하면 절차적으로 코드가 무슨일을 하는지 하나하나 봐야되는 반면, map, forEach, reduce등 선언적인 메소드를 이용하면 어떤 결과가 나올지 바로 코드에서 예측가능해집니다!

Reply:
넵 알겠습니다, 다음부턴 for 문 말고 다른걸로 해보겠습니다!!!